packages/@ama-sdk/schematics/schematics/typescript/core/index.ts

```diff
@@ -260,10 +260,13 @@ function ngGenerateTypescriptSDKFn(options: NgGenerateTypescriptSDKCoreSchematic
       const openApiToolsPath = path.posix.join(targetPath, 'openapitools.json');
       if (tree.exists(openApiToolsPath)) {
         const openApiTools: any = tree.readJson(openApiToolsPath);
-        Object.keys(openApiTools['generator-cli']?.generators)
-          .filter((key) => !openApiTools['generator-cli'].generators[key].inputSpec)
-          .forEach((key) => openApiTools['generator-cli'].generators[key].inputSpec = `./${defaultFileName}`);
-        tree.overwrite(openApiToolsPath, JSON.stringify(openApiTools, null, 2));
+        const generators = openApiTools['generator-cli']?.generators;
```

Comment:
Could be easier to read with 

```suggestion
Object.keys(openApiTools['generator-cli']?.generators || {})
  .filter(...)
  .forEach(...)
```